### PR TITLE
Surya's UserGuide.md part

### LINKED ISSUE #30: Surya's User Guide Section

### PR DESCRIPTION
**Pull request by Tetrerox**
closes #30

### CODE REVIEW

**Review by Gabau — APPROVED** [review]
LGTM.

**Comment by mweeruien** [comment]
LGMT too!

**Review by mweeruien — APPROVED** [review]
LGTM !

File: docs/UserGuide.md [hunk 1]
```diff
@@ -160,37 +160,6 @@ Examples:\
...
-
-Format: 
-* `viewSchedlue [n/name]`
-* `viewSchedlue [id/ID]`
-
-Examples:
-* `viewSchedule n/Candice`
-* `viewSchedule id/12345678`
 
 ### Locating persons by name: `find
```
Comment (Gabau) [inline docs/UserGuide.md:164]: Missing apostrophe.

**Review by Gabau — CHANGES_REQUESTED** [review]
Missing apostrophe.

**Review by Gabau — APPROVED** [review]
LGTM.

**Pull request merged**